components/collective-page/sections/Budget.js

```diff
@@ -1,114 +1,189 @@
+  const isLoading = !allItems.length && data?.loading;
+
+  // Refetch data when used logs in to refresh permissions
   React.useEffect(() => {
     refetch();
   }, [LoggedInUser]);
 
   return (
     <ContainerSectionContent pb={4}>
+      {Boolean(expenses.length && transactions.length) && (
+        <Flex mb={3} flexWrap="wrap" justifyContent="space-between" alignItems="center" maxWidth={720}>
+          <StyledFilters filters={FILTERS} getLabel={geFilterLabel} selected={filter} onChange={setFilter} />
+          <ViewAllLink collective={collective} filter={filter} />
+        </Flex>
+      )}
       <Flex flexDirection={['column-reverse', null, 'row']} justifyContent="space-between" alignItems="flex-start">
-        {isEmpty(data?.transactions) && (
-          <MessageBox type="info" withIcon maxWidth={800} fontStyle="italic" fontSize="14px">
-            <FormattedMessage
-              id="SectionBudget.Empty"
-              defaultMessage="No transaction or expense created yet. They'll start appearing here as soon as you get your first
-                  financial contributors or when someone creates an expense."
-            />
-          </MessageBox>
-        )}
-
         <Container flex="10" mb={3} width="100%" maxWidth={800}>
-          <GraphQLContext.Provider value={budgetQueryResult}>
-            <TransactionsList
-              collective={collective}
-              transactions={data?.transactions?.nodes}
-              displayActions
-              onMutationSuccess={() => refetch()}
-            />
-          </GraphQLContext.Provider>
-          <Flex flexWrap="wrap" justifyContent="space-between" mt={3}>
-            <Box flex="1 1" mx={[0, 2]}>
-              <Link route="transactions" params={{ collectiveSlug: collective.slug }}>
-                <StyledButton
-                  data-cy="view-all-transactions-btn"
-                  my={2}
-                  minWidth={290}
-                  width="100%"
-                  buttonSize="small"
-                  fontSize="14px"
-                >
-                  <FormattedMessage id="CollectivePage.SectionBudget.ViewAll" defaultMessage="View all transactions" />
-                </StyledButton>
-              </Link>
-            </Box>
-            <Box flex="1 1" mx={[0, 2]}>
-              <Link route="expenses" params={{ collectiveSlug: collective.slug }}>
-                <StyledButton
-                  data-cy="view-all-expenses-btn"
-                  my={2}
-                  minWidth={290}
-                  width="100%"
-                  buttonSize="small"
-                  fontSize="14px"
-                >
+          <StyledCard>
+            {isLoading ? (
+              <LoadingPlaceholder height={300} />
+            ) : !allItems.length ? (
+              <Container textAlign="center" py={94} px={2}>
+                <Image src="/static/images/empty-jars.png" alt="Empty jars" width={125} height={125} />
```

Comment:
I'm experimenting with `next/Image` here: https://nextjs.org/docs/api-reference/next/image, it seems to work fine.